clients/drcachesim/tests/schedule_stats_test.cpp

```diff
@@ -280,12 +281,55 @@ test_idle()
     return true;
 }
 
+static bool
+test_cpu_footprint()
+{
+    static constexpr int64_t PID_X = 0;
```

Comment:
May be better to use larger pids, otherwise the reader may conflate workload ordinals (which are used in keys in the impl) with process ids.

Reply:
Agreed, this aligns w/ the other thread on the key vs the stored core identifier.  Done.

Reply:
Err, strike that "aligns" comment, this is the workload ordinal, not the core.